gammapy/estimators/metadata.py

```diff
@@ -0,0 +1,256 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import logging
+from typing import Optional
+import numpy as np
+from scipy import stats
+from astropy.coordinates import SkyCoord
+from pydantic import ValidationError, validator
+from gammapy.utils.metadata import CreatorMetaData, MetaData
+
+__all__ = ["FluxMetaData"]
+
+SEDTYPE = ["dnde", "flux", "eflux", "e2dnde", "likelihood"]
```

Comment:
The better way to handle these is with an `Enum`, or `Literal`

Reply:
Here is an example how we use `Enum` in other places: https://github.com/gammapy/gammapy/blob/main/gammapy/analysis/config.py#L69